internal/statemachine/classic_states.go

```diff
@@ -1049,6 +1066,15 @@ func (stateMachine *StateMachine) populateClassicRootfsContents() error {
 	var classicStateMachine *ClassicStateMachine
 	classicStateMachine = stateMachine.parent.(*ClassicStateMachine)
 
+	// if we backed up resolv.conf then restore it here
+	if osutil.FileExists(filepath.Join(stateMachine.tempDirs.chroot, "etc", "resolv.conf.tmp")) {
```

Comment:
I know it might feel like a waste as it's only used here, but could we add this as a helper function next to `BackupAndCopyResolvConf`, maybe calling it `RestoreResolvConf()` or something? It would be nice and clear then when there's a function for both operations.

Reply:
I thought about making a helper function for this but decided against it because it's only being used once. It would probably make the code cleaner in hindsight, so I'll go ahead and make that change.